taxonomy-editor-frontend/src/pages/editentry/ListAllOtherProperties.jsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Typography } from "@mui/material";
+
+// Parent component used for rendering info
+// on a stopword, synonym, header or footer
+
+const ListAllOtherProperties = ({ nodeObject, id, setNodeObject }) => {
```

Comment:
could this component be integrated in the `AccumulateAllComponents`?, this one is doing minimal stuff, but having it here you need to go 1 level down just to pass the state    (don't know how much this will change in the future)